app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,48 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.TaskManager;
+import com.codesoom.assignment.models.Task;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+    TaskManager taskManager = new TaskManager();
```

Comment:
항상 `private`을 기본으로 생각해 주세요.
```suggestion
    private TaskManager taskManager = new TaskManager();
```

Reply:
아무런 생각없이 public으로 작성했네요;;

Reply:
의외로 흔한 실수이기도 합니다. 이 정도는 괜찮아요 ㅎㅎ